support/keygen-client

```diff
@@ -20,7 +20,7 @@ OPENSSL_VERSION_REGEX_MAJOR_BACKREF="OpenSSL ([0-9]+).*"
 OPENSSL_VERSION_STRING=$(openssl version)
 OPENSSL_VERSION_MAJOR=$(echo "$OPENSSL_VERSION_STRING" | sed -En "s/$OPENSSL_VERSION_REGEX_MAJOR_BACKREF/\1/p")
 
-if [ "$OPENSSL_VERSION_MAJOR" != "1" ]; then
+if [[ $OPENSSL_VERSION_MAJOR -lt 1 ]]; then
```

Comment:
Maybe we can use something like [this](https://github.com/mendersoftware/integration/pull/2125/files#diff-1c6286e89a54136eb0e9ecf34b13a3064a75e54a52681ecdfa667f354ff71bb0R41-R46) instead?